common/source/docs/common-dshot.rst

```diff
@@ -15,16 +15,19 @@ Ardupilot supports several communication protocols with brushless motor ESCs:
 - Oneshot125 (PWM pulse widths divided by 8)
 - DShot150 at 150kbaud (recommended)
 - DShot300 at 300kbaud
-- DShot600 at 600kbaud
+- DShot600 at 600kbaud (recommended)
 - DShot1200 at 1200kbaud
+- Bi-directional DShot at 150, 300, 600 and 1200kbaud on supported firmware
 
-These are selected by the :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` parameter on Copter or :ref:`Q_M_PWM_TYPE <Q_M_PWM_TYPE>` on QuadPlanes for copter function motors. On Plane, other motors use Normal (PWM) protocol.
+These are selected by the :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` parameter on Copter or :ref:`Q_M_PWM_TYPE <Q_M_PWM_TYPE>` on QuadPlanes for copter function motors. On Plane, other motors use Normal (PWM) protocol. Bi-directional DShot is selected by selecting DShot in :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` and then selecting the required channels through :ref:`SERVO_BLH_BDSMASK <SERVO_BLH_BDMASK>`.
```

Comment:
should be SERVO_BLH_BDMASK

Reply:
right....build did not catch that typo since its the label not link....